docs/includes/pairlists.md

```diff
@@ -369,6 +369,8 @@ As this Filter uses past performance of the bot, it'll have some startup-period
 
 Filters low-value coins which would not allow setting stoplosses.
 
+Namely, pairs are blacklisted if the stop price would be the same as 99% of the stop price due to precision rounding on the exchange.
```

Comment:
Is it "the same as 99%" or "greater than 99%"? I can't remember what it is in the code...

Reply:
The code is
```python
        # Adjust stop-prices to precision
        sp = self._exchange.price_to_precision(pair, stop_price, rounding_mode=ROUND_UP)

        stop_gap_price = self._exchange.price_to_precision(pair, stop_price * 0.99,
                                                           rounding_mode=ROUND_UP)
        logger.debug(f"{pair} - {sp} : {stop_gap_price}")

        if sp <= stop_gap_price:
```
`sp` cannot be less than `stop_gap_price`, can it? This is why I wrote it this way.

In fact I'm not sure I understand this very well and any more clear formulation is wellcome.

Reply:
Like how about, "Namely, pairs are blacklisted if a gap of more than one percent in the stop price would be caused by precision rounding on the exchange"?

Reply:
Or "Namely, pairs are blacklisted if a variance of one percent or more in the stop price would be caused by precision rounding on the exchange, i.e. `rounded_price <= stop_price * 0.99`" ?

Reply:
We "might" wanna add a hint on the pointlessness of this in corellation to shorts.

the initial idea was to avoid coins with a value VERY close to their lower trading boundary - not allowing setting of proper stoploss.
this can't happen on shorts ... and on longs in futures - it should in theory trigger a liquidation first ... otherwise how'd dhat coin liquidate ...? 

---

the .99 multiplication is to do "stoploss - 1%" - which is the default for stop limit orders - and gives your actual stop order "some" wiggle room to drop slightly further (stops might not fill at exactly the price they're triggered at).

Reply:
I added another small commit with the hint.
I'm happy to drop the whole change to `docs/includes/pairlists.md` if it is not needed in the docs.

---

I guess that the PR will be squashed on merge, right? I don't need to squash these and rebase.